app/client/routes.js

```diff
@@ -17,3 +17,13 @@ FlowRouter.route('/protocol/:protocol_id', {
 		});
 	}
 });
+
+FlowRouter.route('/protocol/:protocol_id/pdf', {
+	name: 'pdf',
+	action: function(param, query_params) {
+		BlazeLayout.render('search_template', {
```

Comment:
We should really make a new template/layout for rendering fullscreen PDFs with the search bar intact. Or we should generalize the "search_template" template.